### PR TITLE
ADU Dispatcher for v1 launch

### PR DESCRIPTION
**Pull request by hshaosf**

### CODE REVIEW

File: service/modules/common.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    accela_link = "{accela_url}&capID1={cap1}&capID2={cap2}&capID3={cap3}".format(
+        accela_url=os.environ.get('ACCELA_URL'),
+        cap1=accela_sys_caps[1],
+        cap2=accela_sys_caps[2],
+        cap3=accela_sys_caps[3],
+    )
+    return accela_link
+
+# pylint: disable=unidiomatic-typecheck
+def assert_mock(expected, actual):
```
Comment (josh-chou) [inline service/modules/common.py]: should this go in test_common?

File: service/resources/dispatch_email.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            try:
+                #pylint: disable=line-too-long
+                if 'airtable_record_id' not in data_json or data_json['token'] != os.environ.get('EMAIL_TOKEN'):
+                    raise ValueError(ERROR_401)
+
+                # init airtable
+                airtable_id = data_json['airtable_record_id']
+
+                emails_sent = self.send_submission_email_by_airtable_id(airtable_id)
+                msg = emails_sent
```
Comment (josh-chou) [inline service/resources/dispatch_email.py]: this seems extra.  why not just use emails_sent directly?  or set msg = self.send_sub...?

**Review by josh-chou — APPROVED** [review]

File: service/modules/common.py (hunk 1 above)
Comment (hshaosf) [reply service/modules/common.py]: Thanks. Created a separate file for common functions used in tests. 8f48729671829dfcd2c944dee2ef1c340aa32a61

File: service/resources/dispatch_email.py (hunk 2 above)
Comment (hshaosf) [reply service/resources/dispatch_email.py]: Thanks. Updated in 653f710307747b27d7d20a96f3f1af779bf34714

**Pull request merged**